### PR TITLE
SSO Login landing screen

### PR DESCRIPTION
**Pull request by unicornzero**

### CODE REVIEW

File: app/components/sso-login-panel.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  organizationNameWithDefault: computed('provider', function() {
+    return this.organizationName || 'your organization';
+  }),
+  providerWithDefault: computed('provider', function() {
+    return this.provider || 'your Single Sign-On provider';
+  }),
+  mainText: computed('provider', function() {
+    return `Login through ${this.providerWithDefault}`;
+  }),
+  secondaryText: computed('provider', 'organizationName', 'required', function() {
```
[the remark this answers is not in the document]
Comment (unicornzero) [reply app/components/sso-login-panel.js]: thank you

File: tests/acceptance/sso-login-test.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    describe('uses organization-specific information', function() {
+      functionsCorrectly();
+    });
+  });
+
+  describe('when no params are provided', function() {
+    beforeEach(function() {
+      url = '/auth/sso-login';
+      supportMessage = "Hi! I'd like help logging into my  SSO account.";
```
Comment (cadeParade) [inline tests/acceptance/sso-login-test.js]: These look like they are not used?

Comment (unicornzero) [reply tests/acceptance/sso-login-test.js]: Oh! Not anymore! Yep.

Comment (unicornzero) [reply tests/acceptance/sso-login-test.js]: They're what's used by `functionsCorrectly`

Comment (cadeParade) [reply tests/acceptance/sso-login-test.js]: ah right...could you make them function arguments? Scoping works, but is less explicit.

**Pull request merged**